db/db_setup.sql

```diff
@@ -20,17 +20,17 @@ CREATE TABLE `courses` (
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE utf8mb4_unicode_ci;
 
 CREATE TABLE `users` (
-    `id` INT NOT NULL AUTO_INCREMENT,
-    `username` VARCHAR(80) NOT NULL,
-    `email` VARCHAR(80),
-    `slackuid` VARCHAR(25) NOT NULL UNIQUE,
-    PRIMARY KEY (`id`)
+	`id` VARCHAR(9),
```

Comment:
This is only for sake of consistency right? Since we are removing manual DB setup completely.

Reply:
Correct - I had this just for testing. We will remove this file.